pkg/api/protobuf/go/minder/v1/minder.pb.go

```diff
@@ -15,7 +15,7 @@
 
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.31.0-devel
+// 	protoc-gen-go v1.31.0
```

Comment:
there seems to be a lot of spam in this diff is it related to the different protoc version I have?

Reply:
probably, yes

Reply:
Is there a make target I can use to generate this file?

Reply:
yeah, `make gen` takes care of all generation-related things 👍